src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseNotificationsHandler.java

```diff
@@ -46,11 +46,14 @@ public class HomeOfficeCaseNotificationsHandler implements PreSubmitCallbackHand
                                                          + "homeOfficeNotificationsEligible: {} ";
     private final FeatureToggler featureToggler;
     private final HomeOfficeApi<AsylumCase> homeOfficeApi;
-    private Long cacheCaseId = 99999L;
-    private String cachedEvent;
 
     private static final String HO_NOTIFICATION_FEATURE = "home-office-notification-feature";
 
+    private Cache<Long, Long> cache = Caffeine.newBuilder()

```

Comment:
Issue with inline initialising is that its now hard to test, inject mocks into the class.